web_src/js/components/RepoActionView.vue

```diff
@@ -415,8 +415,8 @@ const sfc = {
       const fullScreenEl = document.querySelector('.action-view-right');
       const outerEl = document.querySelector('.full.height');
       const actionBodyEl = document.querySelector('.action-view-body');
-      const headerEl = document.querySelector('.ui.main.menu');
-      const contentEl = document.querySelector('.page-content.repository');
+      const headerEl = document.querySelector('#navbar');
+      const contentEl = document.querySelector('#actions-content');
```

Comment:
How about just using "page-content" ? It means the main page content.

Reply:
I think the old `'.page-content.repository'` is good enough.

Reply:
reverted